addons/html_builder/static/src/core/builder_options_plugin.js

```diff
@@ -21,6 +21,17 @@ export class BuilderOptionsPlugin extends Plugin {
         clean_for_save_handlers: this.cleanForSave.bind(this),
         post_undo_handlers: this.restoreContainer.bind(this),
         post_redo_handlers: this.restoreContainer.bind(this),
+        // Resources definitions:
+        remove_disabled_reason_providers: [
+            // ({ el, reasons }) => {
+            //     reasons.push(`I hate ${el.dataset.name}`);
```

Comment:
? ^^

Reply:
New concept in the html_builder: documentation